src/webui/api/synccontroller.cpp

```diff
@@ -550,7 +550,10 @@ void SyncController::torrentPeersAction()
             peer[KEY_PEER_COUNTRY] = Net::GeoIPManager::CountryName(pi.country());
         }
 
-        peers[pi.address().ip.toString() + ':' + QString::number(pi.address().port)] = peer;
+        if (pi.address().ip.toString().contains(':'))  // IPv6
```

Comment:
```c++
const QString ip {pi.address().ip.toString()};
const auto port {QString::number(pi.address().port)};
const auto endpoint {QString::fromLatin1(ip.contains(':') ? "[%1]:%2" : "%1:%2").arg(ip, port)};
peers[endpoint] = peer;
```

Reply:
Thank you. It is replaced by `PeerAddress::toString()`, and we don't need to deal with the ip protocol again.